src/styles/style.ts

```diff
@@ -0,0 +1,36 @@
+const style={
```

Comment:
instead of a large style file every component should have a styles file with related styles only.